Fakes/OpenActive.FakeDatabase.NET/DatabaseTables.cs

```diff
@@ -71,6 +71,12 @@ public class OrderItemsTable : Table
 
         public BookingStatus Status { get; set; }
         public decimal Price { get; set; }
+        public AccessCode AccessCode {get; set;}
```

Comment:
```suggestion
        public string PinCode {get; set;}
```

Can just store a pin code here and out and output it as an accessCode